.vscode/tasks.json

```diff
@@ -28,8 +28,8 @@
         "type": "shell",
         "args": [
           "msbuild",
-          "src/Compilers/CSharp/csc/csc.csproj",
-          "-p:UseRoslynAnalyzers=false"
+          "-p:UseRoslynAnalyzers=false",
```

Comment:
Unintentional change?

Reply:
It was deliberate: options before project.